client/src/components/App.js

```diff
@@ -89,6 +90,7 @@ class App extends Component {
               <PrivateRoute path="/settings" authed={authed} />
             </Switch>
           </Layout>
+          <Subscribe />
```

Comment:
We should put this in a component that doesn't persist across all components

Reply:
I need more information about when the user should see this alert in order to know where to place this component

Reply:
Okay last change made